### PR TITLE
Move and rename some tests.

### PR DESCRIPTION
**Pull request by wujingyue**
...because test_resize.cpp is getting too large.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -389,44 +389,45 @@ list(APPEND JIT_TEST_SRCS
   ${NVFUSER_SRCS_DIR}/kernel_db/test/test_nvfuser_kernel_db_open.cpp
   ${NVFUSER_SRCS_DIR}/kernel_db/test/test_nvfuser_kernel_db_query.cpp
   ${NVFUSER_SRCS_DIR}/kernel_db/test/test_nvfuser_kernel_db_write.cpp
+  ${NVFUSER_ROOT}/test/test_alias.cpp
```
Comment (wujingyue) [inline CMakeLists.txt:392]: I sorted the files -- no behavior change.

**Comment by wujingyue** [comment]
!build

File: CMakeLists.txt (hunk 1 above)
Comment (zasdfgbnm) [reply CMakeLists.txt:392]: Just curious: do you do this manually, or there is some tool for doing so?

Comment (wujingyue) [reply CMakeLists.txt:392]: In vim, select the lines and :sort

**Review by zasdfgbnm — APPROVED** [review]

**Review by Priya2698 — APPROVED** [review]

**Comment by wujingyue** [comment]
!build

**Pull request merged**